specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2017-07-01/bms.json

```diff
@@ -0,0 +1,3568 @@
+      "properties": {
+        "featureType": {
+          "description": "backup support feature type.",
+          "type": "string"
+        }
+      },
+      "discriminator": "featureType"
+    },
+    "GenericProtectedItem": {
+      "description": "Base class for backup items.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProtectedItem"
+        }
+      ],
+      "properties": {
+        "friendlyName": {
+          "description": "Friendly name of the container.",
+          "type": "string"
+        },
+        "policyState": {
+          "description": "Indicates consistency of policy object and policy applied to this backup item.",
+          "type": "string"
+        },
+        "protectionState": {
+          "description": "Backup state of this backup item.",
+          "enum": [
+            "Invalid",
+            "IRPending",
+            "Protected",
+            "ProtectionError",
+            "ProtectionStopped",
+            "ProtectionPaused"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ProtectionState",
+            "modelAsString": true
+          }
+        },
+        "protectedItemId": {
+          "format": "int64",
+          "description": "Data Plane Service ID of the protected item.",
+          "type": "integer"
+        },
+        "sourceAssociations": {
+          "description": "Loosely coupled (type, value) associations (example - parent of a protected item)",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "fabricName": {
+          "description": "Name of this backup item's fabric.",
+          "type": "string"
+        }
+      },
+      "x-ms-discriminator-value": "GenericProtectedItem"
+    },
+    "GenericProtectionPolicy": {
+      "description": "Azure VM (Mercury) workload-specific backup policy.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProtectionPolicy"
+        }
+      ],
+      "properties": {
+        "subProtectionPolicy": {
+          "description": "List of sub-protection policies which includes schedule and retention",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubProtectionPolicy"
+          }
+        },
+        "timeZone": {
+          "description": "TimeZone optional input as string. For example: TimeZone = \"Pacific Standard Time\".",
+          "type": "string"
+        },
+        "fabricName": {
+          "description": "Name of this policy's fabric.",
+          "type": "string"
+        }
+      },
+      "x-ms-discriminator-value": "GenericProtectionPolicy"
+    },
+    "Job": {
+      "description": "Defines workload agnostic properties for a job.",
+      "required": [
+        "jobType"
+      ],
+      "type": "object",
+      "properties": {
+        "entityFriendlyName": {
+          "descrip
```

Comment:
Please add description to properties.

Reply:
Description is already present in this case.